services/ingestion/etl/types/Project.js

```diff
@@ -25,16 +26,18 @@ type GeoJSON =
   | MultiPolygon2D;
 
 type Location = {
-  country_name?: string,
   country_code: string,
   region?: string,
   nuts2?: string,
+  address?: string,
+  postal_code?: string,
+  town?: string,
   location: GeoJSON,
 };
 
 type Timeframe = {
-  from: string | null,
-  to: string | null,
+  from: string | '',
```

Comment:
As discussed on slack, no defaults on types :)

Reply:
So should we remove every '| ...' in this file?

Reply:
I think these should also be updated

```
  total_cost?: string | null,
  private_fund?: string | null,
  public_fund?: string | null,
  other_contrib?: string | null,
  funding_area?: string | null,
```

if they were either a string or a number, then we can have unions like `total_cost?: string | number`. At any case, it's about the type of data, not default values, which I must have incorrectly used as well...